src/Chirp.Web/Pages/Public.cshtml.cs

```diff
@@ -37,12 +39,27 @@ public async Task<IActionResult> OnGet([FromQuery] int page)
         
         return Page();
     }
-    public async Task<IActionResult> OnPost(CheepDTO cheepDTO)
+
+    public async Task<IActionResult> OnPost()
     {
         if (User.Identity.IsAuthenticated)
         {
-            var AuthorName = User.Identity.Name;
-            await _service.CreateCheep(cheepDTO, AuthorName);
+            var authorName = User.Identity.Name;
+            var authorEmail = $"{authorName}@example.com"; // Adjust this as needed
```

Comment:
This Is not needed an will create an issue since this will put the Example.com domain to the user name